test/regressions/index.test.ts

```diff
@@ -62,7 +62,7 @@ async function main() {
 
   // Wait for all requests to finish.
   // This should load shared resources such as fonts.
-  await page.goto(`${baseUrl}#no-dev`, { waitUntil: 'networkidle0' });
+  await page.goto(`${baseUrl}#no-dev`, { waitUntil: 'networkidle' });
```

Comment:
This seemed a bug

Reply:
Are you sure? 🤔 
https://www.webshare.io/academy-article/puppeteer-networkidle0-vs-networkidle2
it seems like it is correct. 🤔

Reply:
We are using playwright, the types flag that as a type error

Reply:
Ok, nice. 👍 
It's a difference between Playwright and Puppeteer. 🙈 
https://playwright.dev/docs/api/class-page#page-goto-option-wait-until